lib/process-services-cloud/src/lib/app/services/apps-process-cloud.service.ts

```diff
@@ -80,8 +80,11 @@ export class AppsProcessCloudService {
             );
     }
 
-    private getApplicationUrl(): string {
-        return `${this.appConfigService.get('bpmHost')}/deployment-service/v1/applications`;
+    private getApplicationUrl(envKey?: string): string {
+        let contextRoot = this.appConfigService.get('bpmHost', '');
+        contextRoot = 'https://hxps.dev.hxp.hyland.com';
+        contextRoot = contextRoot.split('://').join(`://${envKey ? 'hxps-' + envKey + '.' : ''}`);
```

Comment:
is it okay that the ADF lib now tightly bound to the "hxps-" ?

Reply:
It shouldn't. Maybe I can add the hxps as part of the string that is sent to ADF, what do you think?